new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+GO_SRC_PATH="$(go env GOPATH)/src"
+CURRENT_BRANCH=$(git branch --show-current)
+
```

Comment:
Execution is brittle to invocation location.  Use `cd $(dirname $0)` to resolve to script dir.

Reply:
Done.